csharp/test/Glacier2/router/Client.cs

```diff
@@ -231,8 +231,8 @@ public override void run(string[] args)
             {
                 Console.Out.Write("testing oneway callback... ");
                 Console.Out.Flush();
-                CallbackPrx oneway = CallbackPrxHelper.uncheckedCast(twoway.ice_oneway());
-                CallbackReceiverPrx onewayR = CallbackReceiverPrxHelper.uncheckedCast(twowayR.ice_oneway());
+                CallbackPrx oneway = twoway.ice_oneway();
```

Comment:
The remainder of this PR is test updates. As you can see it's much more convenient to not uncheckedCast or cast the returned proxy.